packages/x-tree-view/src/TreeItem2DragAndDropOverlay/TreeItem2DragAndDropOverlay.tsx

```diff
@@ -52,9 +58,11 @@ const TreeItem2DragAndDropOverlayRoot = styled('div', {
       style: {
         marginLeft:
           'calc(var(--TreeView-indentMultiplier) * calc(var(--TreeView-itemDepth) - 1))' as any,
-        borderBottom: `1px solid ${alpha((theme.vars || theme).palette.grey[900], 0.6)}`,
+        borderBottom: theme.vars
+          ? `1px solid rgba(${theme.vars.palette.grey[900]} / ${0.6})`
+          : `1px solid ${alpha(theme.palette.grey[900], 0.6)}`,
         ...theme.applyStyles('dark', {
-          borderBottomColor: alpha((theme.vars || theme).palette.grey[900], 0.6),
+          borderBottomColor: theme.vars ? `rgba(${theme.vars.palette.grey[900]} / ${0.6})` : alpha(theme.palette.grey[900], 0.6),
```

Comment:
@flaviendelangle @noraleonte could you double check if this is not an error? 🤔 
It seems that this should maybe use `grey[100]` as in all other cases for the dark theme. 🤔 
Now it does nothing... 🤷